### PR TITLE
Adding 2021 scripts to CondTools/Geometry

### PR DESCRIPTION
**Pull request by AramayisPetrosyan**
#### PR description:

adding 2021 scripts (based on 2019 ones) to the _test_ directory
CondTools/Geometry/test/writehelpers
per request of Ianna Osborne @ianna

#### PR validation:

runTheMatrix.py -s   is OK

#### if this PR is a backport please specify the original PR:

No need for backporting.

### CODE REVIEW

**Comment by abdoulline** [comment]
@ianna and @ggovi 
could you, please, take a look?

**Comment by fabiocos** [comment]
please test

testing nothing in this PR...

File: CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```
Comment (ggovi) [inline CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh]: the conddb_import command will be soon decommissioned. please replace with conddb as follows:
conddb --db [input file] [input_tag] [destination_tag] --destdb [dest_file] 
example:
conddb --db myfile.db XMLFILE_Geometry_TagXX_Extended2021_mc XMLFILE_Geometry_TagXX_Extended2021_mc --destdb GeometryFileExtended2021.db

Comment (ianna) [reply CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh]: @ggovi - this should be done for all the scripts, not just in this PR. IMHO, it should be done at a later stage.

Comment (ggovi) [reply CondTools/Geometry/test/writehelpers/splitExtended2021Database.sh]: @ianna 
true it should be done for all the existing scripts, but a new PR should be already compliant - we should not add something that we already know needs to be modified...

**Comment by AramayisPetrosyan** [comment]
Dear All,
Due to problems with my actual branch, I'm closing this PR and will
submit a new one...
Aramayis

**Pull request closed without merging**